corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/controller.js

```diff
@@ -5,11 +5,17 @@ hqDefine("cloudcare/js/formplayer/apps/controller", function () {
         FormplayerFrontend = hqImport("cloudcare/js/formplayer/app"),
         settingsViews = hqImport("cloudcare/js/formplayer/layout/views/settings"),
         views = hqImport("cloudcare/js/formplayer/apps/views");
+
     return {
         listApps: function () {
-            $.when(FormplayerFrontend.getChannel().request("appselect:apps")).done(function (apps) {
+            $.when(FormplayerFrontend.getChannel().request("appselect:apps")).done(function (appCollection) {
+                let apps = appCollection.toJSON();
```

Comment:
The app object is defined [here](https://github.com/dimagi/commcare-hq/blob/52ce42994013e34fd487a0ac7ffe0b583ba1c0b7/corehq/apps/cloudcare/dbaccessors.py#L17) in the backend and I'm finding an app is approx 50KB by recursively running `sys.getsizeof` on the nested objects. And that the operation takes 1 millisecond to run locally with `console.time`/ `console.timeEnd`, so I don't think it's slow.

Is that the right way of going about checking?

Reply:
The empirical data is good if the local app you're running it on is representative of a big prod app (like, if it's a copy of one of the USH CICT prod apps).

The code you linked is better evidence: because it's pulling from `get_brief_apps_in_domain`, you know the data comes from [this couch view](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/_design/views/applications_brief/map.js) which is a small subset of app properties that won't be large even for big apps (specifically, there's no `modules` attribute).

Reply:
ah, that's a better way to go about it. Also, I found a big app and a new app are basically the same size so that makes sense, thank you